### DISCUSSION TITLE
[Enhancment] Add your Reddit for "r/EasterEggs"

### DISCUSSION THREAD

**Discussion by simsimient1** [hushenghao/AndroidEasterEggs#227] [Ideas]
It's just Easter Eggs and much better added "r/EasterEggs" add settings "About" click button "Reddit Easter Eggs" the other way.

Material Icons:
For example of Reddit Icons, the other Repository Material Icons.

Comments:
@hushenghao and @simsimient1

**Comment by hushenghao** [comment]
<img width="659" alt="image" src="https://github.com/user-attachments/assets/27741e28-9e39-4aae-b341-b50fd707167e">

WTF, stupid Reddit

**Reply by NurMiApp** [reply]
A r/reclassified is flair [Banned]!

**Reply by NurMiApp** [reply]
Oops... Hello @hushenghao???

**Reply by HeySimSimi** [reply]
After bot:
A reupload of @simsimient1 a message this good bot.

*Good bot!*

**Reply by hushenghao** [reply]
Good bye Reddit
